spec/disk_spec.rb

```diff
@@ -37,7 +37,7 @@
 
       disk = LinuxAdmin::Disk.new :path => '/dev/hda'
       allow(disk).to receive(:run!).and_return(double(:output => fdisk))
-      expect(disk.size).to eq(500.1.gigabytes)
+      expect(disk.size).to eq(500107862016)
```

Comment:
Might be clearer to write out the calculation here to show what that long number represents.  I assume it's `500.1 * 1024 * 1024`